release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -472,6 +472,43 @@ For more information about disabling the Image Registry Operator, see xref:../in
 [id="ocp-4-14-olm"]
 === Operator lifecycle
 
+[id="ocp-4-14-olmv1"]
+==== {olmv1-first} (Technology Preview)
+
+Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release. {product-title} 4.14 introduces components for a next-generation iteration of OLM as a Technology Preview feature, known during this phase as _{olmv1}_. This updated framework evolves many of the concepts that have been part of previous versions of OLM and adds new capabilities.
+
+During this Technology Preview phase of {olmv1} in {product-title} 4.14, administrators can explore the following features:
+
+Fully declarative model that supports GitOps workflows::
+{olmv1} simplifies Operator management through two key APIs:
++
+--
+* The `Operator` API, provided by the new Operator Controller component, streamlines management of installed Operators by consolidating user-facing APIs into a single object. This empowers administrators and SREs to automate processes and define desired states by using GitOps principles.
+* The `Catalog` API, provided by the new Catalogd component, serves as the foundation for {olmv1}, unpacking catalogs for on-cluster clients so that users can discover installable content, such as Operators and Kubernetes extensions. This provides increased visibility into all available Operator bundle versions, including their details, channels, and update edges.
+--
++
+For more information, see _Operator Controller_ and _Catalogd_.
```

Comment:
Did you mean to add xrefs here?

Reply:
Sorry, yea, mentioned above in https://github.com/openshift/openshift-docs/pull/63605#issuecomment-1769245183.